analyzer/src/main/kotlin/MavenSupport.kt

```diff
@@ -117,8 +118,9 @@ class MavenSupport(localRepositoryManagerConverter: (LocalRepositoryManager) ->
     private fun createMavenExecutionRequest(): MavenExecutionRequest {
         val request = DefaultMavenExecutionRequest()
 
-        request.systemProperties["java.home"] = System.getProperty("java.home")
-        request.systemProperties["java.version"] = System.getProperty("java.version")
+        val props = System.getProperties()
+        EnvironmentUtils.addEnvVars(props)
+        request.systemProperties = props
```

Comment:
Is it OK to overwrite *everything* in here (as it's empty), or should we merge `props` into `request.systemProperties`?

Reply:
Yes, it's empty before.